testing/py-backcall/APKBUILD

```diff
@@ -8,14 +8,24 @@ url="https://github.com/takluyver/backcall"
 arch="noarch"
 license="BSD-3-Clause"
 makedepends="python2-dev python3-dev"
-options="!check" # pytest broken on alpine
+#options="!check" # pytest broken on alpine
```

Comment:
please remove this line instead of commenting out

Reply:
Yes, forgot to remove it.